config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customers#create"
```

Comment:
This one is tricky. The route and controller action you have here indicates creating a new customer. What we wanted was slightly different, we wanted to create a new item for a specific customer. One option would be to write the route like this: `post "/customers/:id/items", to: "customer_items#create"` so that it's routing to that joins table controller.